### ISSUE TITLE
Broken dependency with later version of mpl_toolkit

### ISSUE THREAD

**Issue by charliekilpatrick** [fforster/DELIGHT#1]
In delight.py, the line:

`from mpl_toolkits.axes_grid.inset_locator import inset_axes`

is broken with the latest version of mpl_toolkit.  Later versions have that function under `mpl_toolkits.axes_grid1.inset_locator` (see: https://matplotlib.org/stable/api/_as_gen/mpl_toolkits.axes_grid1.inset_locator.inset_axes.html).  I recommend updating to:

`from mpl_toolkits.axes_grid1.inset_locator import inset_axes`

**Comment by torna4o** [comment]
> In delight.py, the line:
> 
> `from mpl_toolkits.axes_grid.inset_locator import inset_axes`
> 
> is broken with the latest version of mpl_toolkit. Later versions have that function under `mpl_toolkits.axes_grid1.inset_locator` (see: https://matplotlib.org/stable/api/_as_gen/mpl_toolkits.axes_grid1.inset_locator.inset_axes.html). I recommend updating to:
> 
> `from mpl_toolkits.axes_grid1.inset_locator import inset_axes`

Indeed, I myself changed the "axes_grid1" folder to "axes_grid" hence "Module not found" error for mpl_toolkits.axes_grid disappeared.
